### PR TITLE
Add notice for nebula status

### PR DESCRIPTION
**Pull request by SuperYoko**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Description:
Users find it confusing after start and status service after 3.0.0 where storage port is red blink before add hosts.
Confirmed with @darionyaphet  storaged should start service until is added into cluster.

## How do you solve it?
1. Add notice for storage status if storage port is red blink.
2. Request document modification.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [x] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: scripts/nebula.service [hunk 1]
```diff
@@ -234,6 +234,9 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
+                WARN "[FAQ] ${daemon_name} will start service until add hosts is done."
```
Comment (critical27) [inline scripts/nebula.service:238]: make `add hosts` upper letters? print in red if necessary 😅

File: scripts/nebula.service [hunk 2]
```diff
@@ -234,6 +234,9 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
```
Comment (darionyaphet) [inline scripts/nebula.service:237]: How do you let others know ADD  HOSTS is a command?

Comment (SuperYoko) [reply scripts/nebula.service:237]: Maybe a link to faq in docs?

Comment (darionyaphet) [reply scripts/nebula.service:237]: Maybe a good idea

Comment (Sophie-Xie) [reply scripts/nebula.service:237]: > Maybe a link to faq in docs?

I don‘t think so, it’s a little werid.

File: scripts/nebula.service [hunk 3]
```diff
@@ -234,6 +234,10 @@ function status_daemon {
...
+                WARN "${daemon_name} after v3.0.0 will start service until ${RED}ADD HOSTS${NC}."
```
Comment (pengweisong) [inline scripts/nebula.service:238]: Could just notice the user in plain English: "Add the Storage hosts to a Nebula Graph cluster"?

**Review by critical27 — APPROVED** [review]

File: scripts/nebula.service [hunk 4]
```diff
@@ -234,6 +234,10 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
+                WARN "${daemon_name} after v3.0.0 will not start service until it is added to cluster."
```
Comment (pengweisong) [inline scripts/nebula.service:238]: will not be started?

Comment (SuperYoko) [reply scripts/nebula.service:238]: storaged is started but it hasn't start service. QAQ

**Review by pengweisong — APPROVED** [review]
